hca_ingest/downloader/flattener.py

```diff
@@ -152,8 +152,12 @@ def __get_concrete_process(entity: Entity) -> dict:
 
     @staticmethod
     def __get_concrete_ids(input_entities: Iterable[Entity], core_name: str, id_name: str):
+
+        def sort_by_concrete_type(entity):
+            return entity.schema.concrete_type
         concrete_ids = {}
-        for concrete_type, inputs_iter in groupby(input_entities, lambda entity: entity.schema.concrete_type):
+        for concrete_type, inputs_iter in groupby(sorted(input_entities, key=sort_by_concrete_type),
```

Comment:
@ESapenaVentura could you check if you can update the unit tests for this? I think you can add a test case in `test_flatten_conrete_entity.py` or update the existing test data.